core/js/typespecs/app_types.ts

```diff
@@ -44,6 +44,9 @@ function generateBaseTableSchema(
       .describe(
         `Date of ${objectName} deletion, or null if the ${objectName} is still active`,
       ),
+    updated_at: datetimeStringSchema
```

Comment:
I think we need to update the openapi schema? I'm seeing a new error with the latest main. 
![image](https://github.com/user-attachments/assets/5c150304-93f8-44d3-9054-f0ee81ab59a7)